packages/grafana-ui/src/components/RefreshPicker/RefreshPicker.story.tsx

```diff
@@ -1,70 +1,65 @@
 import { action } from '@storybook/addon-actions';
+import { useArgs } from '@storybook/client-api';
+import { Story } from '@storybook/react';
 import React from 'react';
 
 import { RefreshPicker } from '@grafana/ui';
 
-import { DashboardStoryCanvas } from '../../utils/storybook/DashboardStoryCanvas';
-import { StoryExample } from '../../utils/storybook/StoryExample';
-import { UseState } from '../../utils/storybook/UseState';
 import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
-import { HorizontalGroup } from '../Layout/Layout';
+
+import { Props } from './RefreshPicker';
+import mdx from './RefreshPicker.mdx';
 
 export default {
   title: 'Pickers and Editors/RefreshPicker',
   component: RefreshPicker,
   decorators: [withCenteredStory],
+  parameters: {
```

Comment:
```suggestion
  parameters: {
    controls: {
      sort: 'alpha'
    },
```

adding this will sort our controls alphabetically 😍 see https://storybook.js.org/docs/react/essentials/controls#sorting-controls